controllers/AdminController.php

```diff
@@ -301,16 +316,27 @@ public function actionInfo($id)
      */
     public function actionSwitch($id)
     {
-        $user = $this->findModel($id);
+        $old = Yii::$app->user;
+        $session = Yii::$app->session;
+        $key = self::ORIGINAL_USER_SESSION_KEY;
```

Comment:
I don't think there is a need in this variables. Just use \Yii::$app->user instead of $old, \Yii::$app->session instead of $session and use constant itself instead of $key.